sample/src/main/java/com/deploygate/sample/App.java

```diff
@@ -44,44 +42,34 @@ public void onInitialized(boolean isServiceAvailable) {
                     Log.i(TAG, "SDK is unavailable");
                     DeployGate.logInfo("SDK is unavailable"); // this fails silently
                 }
-            }
-
-            @Override
-            public void onStatusChanged(
-                    boolean isManaged,
-                    boolean isAuthorized,
-                    String loginUsername,
-                    boolean isStopped
-            ) {
+            })
+            .setStatusChangeCallback((isManaged, isAuthorized, loginUsername, isStopped) -> {
                 Bundle bundle = new Bundle();
                 bundle.putBoolean("isManaged", isManaged);
                 bundle.putBoolean("isAuthorized", isAuthorized);
                 bundle.putString("loginUsername", loginUsername);
                 bundle.putBoolean("isStopped", isStopped);
 
-                String message = String.format(Locale.US, "onStatusChanged(%s)", bundle.toString());
+                String message = String.format(Locale.US, "onStatusChanged(%s)", bundle);
 
                 Log.i(TAG, message);
                 DeployGate.logInfo(message);
-            }
-
-            @Override
-            public void onUpdateAvailable(
-                    int revision,
-                    String versionName,
-                    int versionCode
-            ) {
+            })
+            .setUpdateAvailableCallback((revision, versionName, versionCode) -> {
                 Bundle bundle = new Bundle();
                 bundle.putInt("revision", revision);
                 bundle.putString("versionName", versionName);
                 bundle.putInt("versionCode", versionCode);
 
-                String message = String.format(Locale.US, "onUpdateAvailable(%s)", bundle.toString());
+                String message = String.format(Locale.US, "onUpdateAvailable(%s)", bundle);
 
                 Log.i(TAG, message);
                 DeployGate.logInfo(message);
-            }
-        }, true);
+            })
+            .setEnabledOnNonDebuggableBuild(true)
+            .build();
+
+        DeployGate.install(this, configuration);
```

Comment:
use `install(Context, DeployGateSdkConfiguration)` because `install(Application, DeployGateCallback, boolean)` is deprecated.